app/src/main/java/com/xmartlabs/scasas/criminalintent/controller/CrimeController.java

```diff
@@ -27,9 +27,8 @@ public static CrimeController getInstance() {
     return INSTANCE;
   }
 
-  private CrimeController() {
-    generateTestCrimes(100);
-  }
+//  private CrimeController() {
```

Comment:
Never leave commented code (or almost never). It can be retrieved from git history, so it's pointless to keep it.